gnucash/report/reports/standard/advanced-portfolio.scm

```diff
@@ -79,28 +79,16 @@ by preventing negative stock balances.<br/>")
      (gnc:make-multichoice-option
       gnc:pagename-general optname-price-source
       "d" (N_ "The source of price information.") 'pricedb-nearest
-      (list (vector 'pricedb-latest
-		    (N_ "Most recent")
-		    (N_ "The most recent recorded price."))
-	    (vector 'pricedb-nearest
-		    (N_ "Nearest in time")
-		    (N_ "The price recorded nearest in time to the report date."))
-	    )))
+      (list (vector 'pricedb-latest (N_ "The most recent recorded price"))
+            (vector 'pricedb-nearest (N_ "The price recorded nearest in time to the report date")))))
 
     (add-option
      (gnc:make-multichoice-option
       gnc:pagename-general optname-basis-method
       "e" (N_ "Basis calculation method.") 'average-basis
-      (list (vector 'average-basis
-		    (N_ "Average")
-		    (N_ "Use average cost of all shares for basis."))
-	    (vector 'fifo-basis
-		    (N_ "FIFO")
-		    (N_ "Use first-in first-out method for basis."))
-	    (vector 'filo-basis
-		    (N_ "LIFO")
-		    (N_ "Use last-in first-out method for basis."))
-	    )))
+      (list (vector 'average-basis (N_ "Use average cost of all shares for basis"))
+            (vector 'fifo-basis (N_ "Use first-in first-out method for basis"))
+            (vector 'filo-basis (N_ "Use last-in first-out method for basis")))))
```

Comment:
Value names should not be full sentences. Perhaps:
"Average cost of all shares"
"First in, first out"
"Last in, first out"

In any case the "Use " and "for basis" parts should go.